### DISCUSSION TITLE
How can we visualize a Flet application made with Python on an Android device?

### DISCUSSION THREAD

**Discussion by JuanCarlos2021** [flet-dev/flet#1173] [General]
My daughter and I were excited about Flet and finished an Android mobile engineering program. We found out that it can only run on Windows and not on mobile. We have searched the web and YouTube without success. We are very disappointed. We know that we cannot make it standalone as an APK and that this is something that will be done in the future. Also, we are interested in knowing how to use PWA.
So, what can we do now? What is our program for? How could we find a solution to run or view it on the web on Android? Could you give us an example?

**Comment by ndonkoHenri** [comment]
The flet devs are working intensively on bringing flet 'mobile'.
To install a web app as a PWA on mobile or pc is very easy. Just follow the link below which will show you how to install your app as a PWA on every device.
https://mobilesyrup.com/2020/05/24/how-install-progressive-web-app-pwa-android-ios-pc-mac/

Let me know if you have any further issues.
